docs/.vitepress/theme/components/Home.vue

```diff
@@ -125,6 +125,7 @@ const subscribeToNewsletter = async () => {
           :disabled="isBusy"
           id="email"
           type="email"
+          pattern="^[a-zA-Z0-9]+(?:\.[a-zA-Z0-9]+)*@[a-zA-Z0-9]+(?:\.[a-zA-Z0-9]+)*$"
```

Comment:
Can we improve it so that we allow for `name+test@mymail.com` kind of mails?

Reply:
yeah, i just push a fix right before this comment :D it was my mistake, give it another try

Reply:
This is the commit https://github.com/jozu-ai/kitops/pull/539/commits/b550fd01d281bad30f5ca396153cc21547596be0